index.bs

```diff
@@ -1184,8 +1184,36 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
       or APIs but is provides an easier access to some information that can be
       used to fingerprint users.
     </p>
-
-    <section>
+  <section>
+      <h3 id='capabilities-model'>
+        Capabilities Model
+      </h3>
+      <p>
+        This specification supports {{MediaDecodingType}} values of {{file}},
+        {{media-source}} or {{webrtc}} as well as {{MediaEncodingType}}
+        values of {{record}} and {{webrtc}}. 
+      </p>
+      <p>
+        In realtime communications as supported in [[webrtc]], media is
+        transported between peers. Although web sites can play a role in
```

Comment:
I would rename peers as user agents, this might read better to distinguish with websites.
Also s/web sites can play a role/web sites are responsible for/

Reply:
Fixed.